README.md

```diff
@@ -1,17 +1,106 @@
-# Griptape Tool Template
+# Griptape Plugins Template
 
-A Github template repository for creating Griptape tools.
+A Github template repository for creating Griptape plugins.
 
-## Usage
+## Getting Started
 
-This template repository defines the typical structure for a Griptape tool and is meant to be used as a starting point when developing new tools. To create a tool using this template, click the **"Use this template"** button at the top of the repository.
+Via `gh`:
 
-### Steps for creating a new Griptape tool:
+```
+$ gh repo create griptape-plugin-name -p griptape/griptape-plugin-template
+```
+
+Via github web page:
+
+Click on `Use this template`
+
+![](https://docs.github.com/assets/cb-36544/images/help/repository/use-this-template-button.png)
+
+## What is a Griptape Plugin?
+
+A Griptape Plugin is nothing more than a python class that implements one of Griptape's interfaces.
+
+This repository provides a recommended structure for organizing your plugin code, as well as helpful tools for testing and development.
+
+## Plugin Structure
+
+The template repository is structured as follows:
+
+├── griptape
```

Comment:
This wasn't showing as a clean hierarchy in the preview for me - do you want to wrap it in ```?